app/controllers/external_users/claims_controller.rb

```diff
@@ -389,4 +396,16 @@ def present_errors
     @error_presenter = ErrorPresenter.new(@claim)
   end
 
+  def initialize_json_document_importer
```

Comment:
This method, used in the callback above, is the first line of the index action as well. should i remove the line in the index action and rely on the callback?